workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+---
+name: 'Reading the Internet'
+description: 'Use Python to read HTML and extract any information you can find!'
+author: '@RafaelCenzano'
+img: 'https://cloud-p5pdve0kf.vercel.app/2webscrape.jpg'
+---
+
+We can read all the content on websites, but wouldn't it be cool if we could automatically read specific data from websites whenever you wanted? If this interested you, this workshop is for you. I will be showing you how to use webscraping in Python.
```

Comment:
This doesn't do the workshop justice. It's not just a tutorial on webscraping, you're actually building a tool that scrapes COVID case numbers from a website and displays them as text. This paragraph should be edited to more clearly show what you're going to build.